projects/robots/pal_robotics/tiago_steel/protos/TiagoSteel.proto

```diff
@@ -0,0 +1,47 @@
+#VRML_SIM R2019b utf8
+# license: Apache License 2.0.
+# license url: http://www.apache.org/licenses/LICENSE-2.0.
+# documentation url: https://www.cyberbotics.com/doc/guide/tiago-steel
+# This is a PROTO file for Webots for the robot TIAGo (Steel version) from PAL Robotics (Spain).
+# It is composed of the TIAGo base (with Hokuyo URG 04LX_UG01 lidar) and the TIAGo body (torso) with the front arm finished by the parallel gripper.
+
+PROTO TiagoSteel [
+  field  SFVec3f     translation        0 0.095 0
+  field  SFRotation  rotation           1 0 0 -1.5708
+  field  SFString    name               "TIAGo Steel"
+  field  SFString    controller         "tiago_steel"
+  field  SFString    controllerArgs     ""
+  field  SFString    customData         ""
+  field  SFBool      supervisor         FALSE
+  field  SFBool      synchronization    TRUE
+]
+{
+  TiagoBase {
+    translation IS translation
+    rotation IS rotation
+    controller IS controller
+    controllerArgs IS controllerArgs
+    customData IS customData
+    supervisor IS supervisor
+    synchronization IS synchronization
+    name IS name
+    model "PAL Robotics' TIAGo Steel"
+    description "two-wheeled human-like robot with one articulated arm finished by parallel gripper designed by PAL Robotics"
+    bodyExtension [
+      TiagoBody {
+        FrontArmExtension [
+          TiagoFrontArm {
+            endEffector [
+              TiagoGripper {
+              }
+            ]
+          }
+        ]
+      }
+    ]
+    lidarExtension [
+      HokuyoUrg04lxug01 {
+      }
```

Comment:
It may be interesting to keep this as a field (with the `HokuyoUrg04lxug01` by default).

Reply:
:+1: